### PR TITLE
a try to build on github actions

### PR DESCRIPTION
**Pull request by isopov**
Github actions seems much faster than Travis (maybe we should invest some time in optimizing travis build).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+#        go: [1.8, 1.9, '1.10']
```
Comment (PeterIvanov) [inline .github/workflows/ci.yml:12]: Trial and error leftovers?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        go: [1.11, 1.12, 1.13, 1.14, 1.15, 1.16]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go ${{ matrix.go }}
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go }}
+#    - name: "Setup dependencies"
+#      if: ${{ matrix.go == '1.8' }}
```
Comment (PeterIvanov) [inline .github/workflows/ci.yml:22]: So the tests are currently not run against earlier go versions? No way to fix this?

**Review by PeterIvanov — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (isopov) [reply .github/workflows/ci.yml]: Was going to left it for the next PR, but fixed now.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (isopov) [reply .github/workflows/ci.yml]: Fixed

**Review by PeterIvanov — APPROVED** [review]
Great, thanks, Ivan! 👍

**Pull request merged**